Dockerfile.loadtester

```diff
@@ -10,6 +10,9 @@ RUN HELM3_VERSION=3.11.0 && \
 curl -sSL "https://get.helm.sh/helm-v${HELM3_VERSION}-linux-${TARGETARCH}.tar.gz" | tar xvz && \
 chmod +x linux-${TARGETARCH}/helm && mv linux-${TARGETARCH}/helm /usr/local/bin/helm
 
+RUN curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" && \
```

Comment:
This is not OK, as this image is multi-arch not AMD64. Please see https://github.com/fluxcd/flux2/blob/main/Dockerfile on how to correctly set this, also the version must be pinned, like in Flux.

Reply:
> This is not OK, as this image is multi-arch not AMD64. Please see https://github.com/fluxcd/flux2/blob/main/Dockerfile on how to correctly set this, also the version must be pinned, like in Flux.

Thanks for review, I fix it in new PR https://github.com/fluxcd/flagger/pull/1485.
plz check